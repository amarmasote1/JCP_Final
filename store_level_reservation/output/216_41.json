{
    "warnings": null,
    "message": "Success",
    "nodeType": null,
    "body": null,
    "key": "ItemInventoryKey(itemID=17112343050)",
    "status": 200
}