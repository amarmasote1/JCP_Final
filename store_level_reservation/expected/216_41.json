{
    "warnings": null,
    "message": "Success",
    "body": null,
    "key": "ItemInventoryKey(itemID=17112343050)",
    "status": 200
}